controllers/updateservice_controller.go

```diff
@@ -389,6 +391,19 @@ func (r *UpdateServiceReconciler) ensureDeployment(ctx context.Context, reqLogge
 		containers[i].LivenessProbe = original.LivenessProbe
 		containers[i].ReadinessProbe = original.ReadinessProbe
 	}
+	initContainers := updated.Spec.Template.Spec.InitContainers
+	for i := range initContainers {
+		var original *corev1.Container
+		if initContainers[i].Name == "graph-data" {
+			original = resources.graphDataInitContainer
+		} else {
+			reqLogger.Info("encountered unexpected init container in pod", "Container.Name", initContainers[i].Name)
```

Comment:
why are we keeping these?  Can we stomp them off?

Reply:
like:

```go
updated.Spec.Template.Spec.Initcontainers = []whateverType{resources.graphDataInitContainer}
```

or some such?

Reply:
The Cincinnati Deployment is our operand.  We have opinions about its init containers.  Other folks might have opinions too, and attempt to inject additional init containers.  But trying to share ownership of a single Deployment is sticky, and in the CVO we've had many, many bugs as a result of initially attempting a sharing approach (openshift/cluster-version-operator#830, openshift/cluster-version-operator#804, openshift/cluster-version-operator#674, openshift/cluster-version-operator#654, openshift/cluster-version-operator#650, openshift/cluster-version-operator#563, openshift/cluster-version-operator#549 , ...), and no reports that I can remember of any users benefiting from the sharing.